ElectronClient/gui/MainScreen.jsx

```diff
@@ -99,7 +99,18 @@ class MainScreenComponent extends React.Component {
 		const createNewNote = async (template, isTodo) => {
 			const folderId = Setting.value('activeFolderId');
 			if (!folderId) return;
-
+			if (!template) {
+				const templateType = isTodo ? 'defaultTodo' : 'defaultNote';
+				const settingsLabel = Setting.value(templateType);
+				if (settingsLabel != 0) {
+					const matchLabel = this.props.templates.find(i => i.label == settingsLabel);
+					if (!matchLabel) {
+						Setting.setValue(templateType, '0');
+					} else {
+						template = matchLabel.value;
+					}
+				}
+			}
```

Comment:
This ought to be simpler:

```javascript
if (!template) {
    template = Setting.value(isTodo ? 'defaultTodo' : 'defaultNote');`
}
```

Please don't set any setting in here, as we don't want side-effects.

Reply:
I realized the Setting.setValue isn't necessary. I put it there for safety, thinking the set defaultTodo/defaultNote options might remain after refreshing the templates. They don't, so i'll remove it.

I'll also cut down line 103 and 104 to one line like you propose. Only the template filename is stored in Setting.value(default****), not the contents, so we still need to get the contents from props. This change was made as Caleb realized storing the template contents in options won't allow the user to edit a template without losing the default.

Reply:
Sorry, i was wrong about the first part. The settings do remain even after refresh, which leads to a problem when deleting templates and refreshing. I'll see what i can do.